packages/arb-token-bridge-ui/src/components/common/TokenSymbolWithExplorerLink.tsx

```diff
@@ -0,0 +1,95 @@
+import { useMemo } from 'react'
+import * as Sentry from '@sentry/react'
+
+import { ERC20BridgeToken } from '../../hooks/arbTokenBridge.types'
+import {
+  NativeCurrencyErc20,
+  useNativeCurrency
+} from '../../hooks/useNativeCurrency'
+import { isTokenUSDC, sanitizeTokenSymbol } from '../../util/TokenUtils'
+import { ExternalLink } from './ExternalLink'
+import { useNetworks } from '../../hooks/useNetworks'
+import { useNetworksRelationship } from '../../hooks/useNetworksRelationship'
+
+const createBlockExplorerUrlForToken = ({
+  explorerLink,
+  tokenAddress
+}: {
+  explorerLink: string | undefined
+  tokenAddress: string | undefined
+}): string | undefined => {
+  if (!explorerLink) {
+    return undefined
+  }
+  if (!tokenAddress) {
+    return undefined
+  }
+  try {
+    const url = new URL(explorerLink)
+    url.pathname += `token/${tokenAddress}`
+    return url.toString()
+  } catch (error) {
+    Sentry.captureException(error)
+    return undefined
+  }
+}
+
+const isERC20BridgeToken = (
+  token: ERC20BridgeToken | NativeCurrencyErc20 | null
+): token is ERC20BridgeToken =>
+  token !== null && !token.hasOwnProperty('isCustom')
+
+export function TokenSymbolWithExplorerLink({
+  token,
+  tokenSymbolOverride,
+  isParentChain
```

Comment:
Can we pass the `chainId` instead?

Reply:
as discussed on Slack, `isParentChain` is needed here
![image](https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/28a1e6eb-1767-4935-a423-bb4a12459126)